docs/configuration/http_conn_man/route_config/route.rst

```diff
@@ -300,3 +304,24 @@ runtime_key_prefix
 
   **Note:** If the sum of runtime weights exceed 100, the traffic splitting behavior
   is undefined (although the request will be routed to one of the clusters).
+
+.. _config_http_conn_man_route_table_opaque_config:
+
+Opaque Config
+-------------
+
+Additional configuration can be provided to filters through the "Opaque Config" mechanism.  A
+list of Key/Value pairs are specified in the route config.  The configuration is uninterpreted
+by envoy and can be accessed within a user-defined filter.
+
+.. code-block:: json
+
+  [
+    {"name": "...", "value": "..."}
```

Comment:
If there is a way to do it in the schema, I agree that a JSON dictionary of strings is better. I'm just not sure how to do it. If the above works then that's fine.